concord_cli/kill_task.py

```diff
@@ -1,49 +1,184 @@
 #!/usr/bin/env python
+import sys
 import thrift
 import json
 import logging
-from optparse import OptionParser
+import argparse
 from concord_cli.utils import *
+from terminaltables import AsciiTable
 
 logging.basicConfig()
 logger = logging.getLogger('cmd.kill_task')
 logger.setLevel(logging.INFO)
 
 def generate_options():
```

Comment:
Honestly i'm down just to read the json file... any thoughts ? 

If so we would get to delete this. Obviosly the only required fields from the json file would be zookeeper and zk path.